### PR TITLE
Added secret check for GitHub hook

### LINKED ISSUE #53: Validate webhooks against a secret
# Overview

Depends on #70 

As part of the creation of the webhook we need to pass a secret key that we'll need to store somewhere. This allows us later on when receiving the payload from Github to validate that it was actually sent from Github and not from some random POST request.

# Tasks

- [ ] add `GITHUB_SECRET` env var
- [ ] set manually it to repos for now
- [ ] check secret matches webhook payload secret or reject request
- [ ] create issue for rebasing on secure dynamic approach to have one secret per repo in db (proposed) - FOR BETA

### PR DESCRIPTION
**Pull request by roll**
- fixes #53

### CODE REVIEW

**Comment by roll** [comment]
@amercader 
please take a look

File: .travis.yml [hunk 1]
```diff
@@ -51,6 +52,3 @@ deploy:
   username: ekarev
   password:
     secure: YZDt5MrYCoHjhAlHI4uCqoY4bi4Jw/qoh0PdYT1RQS9pJGGCs1K1EIESCRBkqGzECx450fLGUzIpnJkcwRo/AIkvQl/OimUrBDRXTzyNk29lrXOcphN41PMkeUdBzXmwB5wP12ugOSlzIebNGgyyDRWBGMbc9xsu78TxcqaFun3sgaGemHCWUFoAWFdBhIA2OCv0eHBNUMzugd0Z2Faktj5IbxVIAID1m6c2jgjwHH9z6a6wdrP8YSVpg1vDLkzqKi/9lE+DG3q1AhHjEcdRImO+PlcglfjxspLzSGQ8QbQ+uDypNmI+0j6O+bYxcQJVL0yXRcvCFHSWYw6Bw/+TJ0UMY8e6Ifw8cknyqTqkvmqZfdhKF8+8uVOEYiKG/B6bjMjGTuVFQ7BAKYx6ulXfhBAyfhRNgMD3bGvRAvU3q8J9Fs8TI3ER3ndFN1h/yTPot5Y6Mf3ih9pDt9ut7MVc4F2XJ8hMdCT0pCWmOe7ugJZD1CqZbj0BinAcdGSpNE2LpgUX3PAg5bARuL2ds4v9J2oIrp+C7UX1est7C+Wqep5z6irNAnj462sDKx8qzlLwOYSMcPtI0XglRJuSlXzhMUQFugrhxIIwZplp/gkN4+nsmNaRfHVy3tC7rdNRLZuZjHBLXC3arXdOPydoRyGP1A0M2hNWmkkZTJVzf69MFfc=
-
-after_deploy:
-  - deis run 'make migrate'
```
Comment (amercader) [inline .travis.yml]: Just curious, why was that needed?

Comment (roll) [inline .travis.yml]: It didn't work anyway)
